crates/rustc_codegen_spirv/src/linker/inline.rs

```diff
@@ -394,10 +417,17 @@ impl Inliner<'_, '_> {
             post_call_block_insts.insert(
                 0,
                 Instruction::new(
-                    Op::Load,
+                    Op::Phi,
```

Comment:
The structurizer does not support phis, and so this cannot be done. mem2reg should take care of the load/store.